spec/support/steps/a_show_steps_controller.rb

```diff
@@ -9,7 +9,8 @@
 
     context 'when application is found' do
       let(:existing_case) do
-        Claim.create(office_code: 'AAA', defendants: [Defendant.new(main: true, full_name: 'Nigel', maat: '123')])
+        Claim.create(office_code: 'AAA', defendants: [Defendant.new(main: true, full_name: 'Nigel', maat: '123')],
+                     plea: PleaOptions.values)
```

Comment:
is this required?

Reply:
without this, tests were failing for the check answers controller. we need a better solution, discussed with @ivanELEC and this is what we added to make the tests work. it is all related to the comment you did in the 'case_disposal_card.rb' I guess.